cli/azd/pkg/tools/azcli/spring_app.go

```diff
@@ -0,0 +1,49 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appplatform/armappplatform"
+)
+
+type AzCliSpringAppProperties struct {
+	HostNames []string
+}
+
+func (cli *azCli) GetSpringAppProperties(
+	ctx context.Context,
+	subscriptionId, resourceGroup, appName string,
+) (*AzCliSpringAppProperties, error) {
+	client, err := cli.createSpringAppClient(ctx, subscriptionId)
+	if err != nil {
+		return nil, err
+	}
+
+	// use "default" app name
+	springApp, err := client.Get(ctx, resourceGroup, appName, "default", nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed retrieving spring apps properties: %w", err)
+	}
+
+	return &AzCliSpringAppProperties{
+		HostNames: []string{*springApp.Properties.Fqdn},
```

Comment:
Do all Auzre Spring Apps have a Fully Qualified Domain Name assigned?  If not, does the data plane return an empty string for this property?  I'm just trying to understand if this property will ever be `nil`?

Reply:
It won't return nil whether assign public endpoint in my local test env.